client/components/SurrenderForm.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState, useEffect } from 'react'
+import ErrorList from "./ErrorList"
+import _ from 'lodash'
+
+const SurrenderForm = props => {
+  const [successfulSubmit, setSuccessfulSubmit] = useState(false)
+  const [errors, setErrors] = useState({})
+  const [petTypes, setPetTypes] = useState([])
+  const [formData, setFormData] = useState({
+    name: "",
+    phoneNumber: "",
+    email: "",
+    petName: "",
+    petAge: "",
+    petTypeId: "",
+    imgUrl: "",
+    vaccinationStatus: ""
+  })
+
+  useEffect(() => {
+    const fetchPetTypes = async () => {
+      try {
+        const response = await fetch("/api/v1/pets")
+        if (!response.ok) {
+          const errorMessage = `${response.status} (${response.statusText})`
+          const error = new Error(errorMessage)
+          throw (error)
+        }
+        const petTypeData = await response.json()
+        setPetTypes(petTypeData.petTypes)
+      } catch (err) {
+        console.error(`Error in fetch: ${err.message}`)
+      }
+    }
+    fetchPetTypes()
+  }, []) 
+
+  const petTypeOptions = petTypes.map(petType => {
+    let typeName = petType.type.charAt(0).toUpperCase() + petType.type.slice(1,-1)
+    return (
+      <option key={petType.id} value={petType.id}>
+        {typeName}
+      </option>
+    )
+  })  
+
+
+  const vaxStatuses = {
+    '': '',
+    'Vaccinated': true,
+    'Not Vaccinated': false
+  }
+
+  const vaxStatusOptions = Object.keys(vaxStatuses).map(status => {
+    return (
+      <option key={vaxStatuses[status]} value={vaxStatuses[status]}>
+        {status}
+      </option>
+    )
+  })
+
+  const postNewPet = async (formPayload) => {
+    try {
+      const response = await fetch("/api/v1/adoptions/new", {
+        method: 'POST',
+        headers: new Headers({
+          'Content-Type': 'application/json'
+        }),
+        body: JSON.stringify(formPayload)
+      })
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw (error)
+      }
+      setSuccessfulSubmit(true)
+    } catch (err) {
+      console.error(`Error in fetch: ${err.message}`)
+    }
+  }
+
+  const validForSubmission = () => {
+    let submitErrors = {}
+    const requiredFields = ["name", "phoneNumber", "email", "petName", "petAge", "petTypeId", "imgUrl", "vaccinationStatus"]
+    const displayedFields = ["Name", "Phone Number", "E-mail", "Pet Name", "Pet Age", "Pet Type", "Image URL", "Vaccination Status"]
```

Comment:
One step better: we could use an _object_ here to tie a field name to its display name.

Even better than that: we can use `lodash`'s `startCase`! 
https://lodash.com/docs/4.17.15#startCase